docs/reference/base-specification.md

```diff
@@ -0,0 +1,126 @@
+
+For practical purposes, we have defined a non-normative recommendation for how base files should be defined. _Non-normative_ means: the ontology developer may decide to adopt, modify or alter this recommendation in any way they want, as long as the _normative_ constraints above are not violated.
+
+A "base file" SHOULD:
+
+- include all "base entities", including deprecated ones
+- include all standard and any number of user defined annotations on "base entities". Standard annotations are defined by the [Ontology Metatada Ontology (OMO)](https://github.com/information-artifact-ontology/ontology-metadata) and include labels (rdfs:label), deprecation assertions (owl:deprecated) and definitions (IAO:0000115). User-defined annotations are all annotations that are non-standard and are intended for inclusion in the public release.
+- include a non-redundant set of reasoner-inferred axioms, including
+   - inferred `rdfs:subClassOf` axioms between "base entities"
+   - inferred `rdfs:subClassOf` axioms from "base entities" to non-base entity parents
+   - inferred "existential restrictions", axioms of the form "`:A rdfs:subClassOf :R some :B`" where `:A` is a "base entity", and `:R` is an object property as defined by the ontology developer responsible for defining the release.
+- "non-redundant" means that the `rdfs:subClassOf` sub-graph of the ontology corresponds to the "transitive reduct". In laypersons terms this means, roughly, that if a `rdfs:subClassOf` axiom is already implied by a path of `rdfs:subClassOf` axioms, that axiom is considered "redundant". (For example: given `:A rdfs:subClassOf :B`, `:B rdfs:subClassOf :C`, `:A rdfs:subClassOf :C`, the latter, `:A rdfs:subClassOf :C` is redundant because it is already implied by tge path through the former two axioms.)
+- break down all `owl:equivalentClass` axioms into `rdfs:subClassOf` axioms where the subject is a "base entity"
+
+## Implementation
+
+This is the current implementation of the recommended base technical specification above. The full operational definition of the process for generating the "base file" is:
+
+```
+# base: A version of the ontology that does not include any externally imported axioms.
+$(ONT)-base.owl:
+	robot merge -i $(SRC) -i $(OTHER_SRC) -i $(ALL_IMPORTS) \
+	reason --reasoner ELK --equivalent-classes-allowed asserted-only --exclude-tautologies structural \
+	relax \
+	reduce -r ELK \
+	remove --base-iri $(URIBASE)/OBA --axioms external --preserve-structure false --trim false \
+    {STANDARD_COMMANDS}
+	--output $@
+```
+
+_This is a breakdown of the sub-processes:_
+
+- `robot merge -i $(SRC) -i $(OTHER_SRC) -i $(ALL_IMPORTS)`:  Merge the ontology with all its dependencies together.
+- `reason --reasoner {REASONER} --equivalent-classes-allowed asserted-only --exclude-tautologies structural`: Add all inferred `rdfs:subClassOf` axioms.
+- `relax`: break down all `owl:equivalentClass` axioms into `rdfs:subClassOf` axioms where the subject is a "base entity"
```

Comment:
Maybe delete equivlaent class axioms between names prior to relaxing to avoid things like MOndo subclassOf DOI